fix: remove override content path and update doc (#684)
AmadeusITGroup/otter: packages/@o3r/dynamic-content/README.md

export class MyModule {}
```

## getContentPathStream

For non-media resources (ex: localization, configuration) one should refer to the ``getContentPath`` method of the ``DynamicContentService``. Doing so will ensure the correct resource path is computed in any environment (e.g. when the ``data-dynamiccontentpath`` tag is present in the body).

The content path is always related to the root of the application.

> It also ignores any path overrides from the AssetPathOverrideStore store, meaning that you will always get the same file.

Reviewer: ```suggestion
It also ignores any path overrides from the `AssetPathOverrideStore` store, meaning that you will always get the same file.
```

```typescript
@Component({
  /** */
})
export class MyComponent {
  constructor(private service: DynamicContentService) {}

>   async getDynamicConfig() {
>     const filePath = await firstValueFrom(this.service.getContentPathStream('global.config.post.json'));
>     const fileContent = await fetch(filePath);
>     //  ...
>   }

Reviewer: I would have the same argument as the PR #679 :
I understand that you wanted to provide a simple example but it is not a recommended way to use the service (to support the rules engine).
To be more aligned with the guideline, it would be:

```suggestion
  public dynamicConfig$ = this.service.getContentPathStream('global.config.post.json').pipe(
    switchMap((filePath) => from(fetch(filePath))
  );
```

  constructor(private service: DynamicContentService) {}

  async getDynamicConfig() {
    const filePath = await firstValueFrom(this.service.getMediaPathStream('imgs/my-image.png'));
    const fileContent = await fetch(filePath);
    //  ...
  }
}
```

> It also looks for overrides in the AssetPathOverrideStore, so it will return the new value instead if an override is found.

Reviewer 2: ```suggestion
It also looks for overrides in the `AssetPathOverrideStore`, so it will return the new value instead if an override is found.
```
